test/v1/connectors/AmendDisclosuresConnectorSpec.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.connectors
+
+import mocks.MockAppConfig
+import uk.gov.hmrc.domain.Nino
+import v1.mocks.MockHttpClient
+import v1.models.domain.DesTaxYear
+import v1.models.outcomes.ResponseWrapper
+import v1.models.request.disclosures.{AmendDisclosuresRequest, AmendDisclosuresRequestBody, AmendTaxAvoidance}
+
+import scala.concurrent.Future
+
+class AmendDisclosuresConnectorSpec extends ConnectorSpec {
+
+  private val nino: String = "AA111111A"
+  private val taxYear: String = "2019"
+
+  val amendTaxAvoidance: AmendTaxAvoidance = AmendTaxAvoidance("12345","2020-12")
+
+  val amendDisclosuresRequest: AmendDisclosuresRequest = AmendDisclosuresRequest(
+    nino = Nino(nino),
+    desTaxYear = DesTaxYear(taxYear),
+    body = AmendDisclosuresRequestBody(Some(Seq(amendTaxAvoidance)))
+  )
+
+  class Test extends MockHttpClient with MockAppConfig {
+
+    val connector: AmendDisclosuresConnector = new AmendDisclosuresConnector(
+      http = mockHttpClient,
+      appConfig = mockAppConfig
+    )
+
+    MockedAppConfig.desBaseUrl returns baseUrl
+    MockedAppConfig.desToken returns "des-token"
+    MockedAppConfig.desEnvironment returns "des-environment"
+  }
+
+  "AmendDisclosuresConnector" when {
+    "amendDisclosures" must {
+      "return a 204 status for a success scenario" in new Test {
+        val outcome = Right(ResponseWrapper(correlationId, ()))
+
+        MockedHttpClient
+          .put(
+            url = s"$baseUrl/some-placeholder/disclosures/$nino/$taxYear",
```

Comment:
wrong url

Reply:
What is the uri? I can't see one in the Jira story.